tensorflow/tools/git/gen_git_source.py

```diff
@@ -112,9 +112,14 @@ def configure(src_base_path, debug=False):
     if src is None:
       open(os.path.join(gen_path, target), "w").write("")
     else:
-      if hasattr(os, 'symlink'):
-        os.symlink(src, os.path.join(gen_path, target))
-      else:
+      try:
+        # In python 3.5, symlink function exists even on Windows
+        # But requires previliaged right, otherwise a OSError will be thrown
```

Comment:
IIRC, Windows requires a special user right for creating symbolic links. It's available if you have Administrator privileges, and although it can be extended to regular users, it isn't by default....